s3/pkg/service/object.go

```diff
@@ -375,13 +375,14 @@ func (s *s3Service) GetObject(ctx context.Context, req *pb.GetObjectInput, strea
 
 		if bucket.ServerSideEncryption.SseType == "SSE" {
 			// decrypt and write
-			decErr, decBytes := utils.DecryptWithAES256(buf[0:n], bucket.ServerSideEncryption.EncryptionKey)
+			decErr, decBytes := utils.DecryptWithAES256(buf[0:left], bucket.ServerSideEncryption.EncryptionKey)
 			if decErr != nil {
 				log.Errorln("failed to decrypt data. err:", decErr)
 				return decErr
 			}
 			log.Infoln("successfully decrypted")
 			buf = decBytes
+			log.Infoln(string(decBytes))
```

Comment:
Should this be logged? If yes, then please add message to in the log

Reply:
sheesh !!! missed removing debug logs, these are removed. thanks